### PR TITLE
Bug 2043556: Add ci-operator config

### PR DESCRIPTION
**Pull request by elfosardo**

### CODE REVIEW

**Comment by elfosardo** [comment]
this will need to be backported to 4.10

File: .ci-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+build_root_image:
+  name: release
+  namespace: openshift
+  tag: golang-1.15
```
Comment (andfasano) [inline .ci-operator.yaml:4]: What about using `rhel-8-release-golang-1.17-openshift-4.10` as tag, as it seems pretty used also in other repos?

Comment (elfosardo) [reply .ci-operator.yaml:4]: I believe this is a dummy config anyway as it's replaced by whatever we have in the release repo configuration for each single image
But yeah, it probably makes more sense, we'll have to change it in ironic-image too for consistency

Comment (elfosardo) [reply .ci-operator.yaml]: @andfasano https://github.com/openshift/ironic-image/pull/252

**Pull request merged**